spinn_front_end_common/mapping_algorithms/on_chip_router_table_compression/mundy_on_chip_router_compression.py

```diff
@@ -49,15 +49,15 @@ class MundyOnChipRouterCompression(object):
 
     def __call__(
             self, routing_tables, transceiver, machine, app_id,
-            provenance_file_path, compress_only_when_needed=True,
+            system_prov_folder, compress_only_when_needed=True,
```

Comment:
All the changes in this file rename the parameter, but otherwise change nothing.  I don't think the old name was that bad...

Reply:
explictness. its not using the prov file path. its specifically using the system one, as its a system operation.

Reply:
OK, not that critical